Formula/ntfs-3g.rb

```diff
@@ -1,9 +1,20 @@
 class Ntfs3g < Formula
   desc "Read-write NTFS driver for FUSE"
   homepage "https://www.tuxera.com/community/open-source-ntfs-3g/"
-  url "https://tuxera.com/opensource/ntfs-3g_ntfsprogs-2017.3.23.tgz"
-  sha256 "3e5a021d7b761261836dcb305370af299793eedbded731df3d6943802e1262d5"
-  revision 1
+  revision 2
+  stable do
+    url "https://tuxera.com/opensource/ntfs-3g_ntfsprogs-2017.3.23.tgz"
+    sha256 "3e5a021d7b761261836dcb305370af299793eedbded731df3d6943802e1262d5"
+
+    # Detection of struct stat members fails Xcode 9
```

Comment:
That's a very old unsupported version. Please make this a normal patch block

Reply:
Thanks for suggestion, this code also smells bad for me. Could you please give me example of stable-only patch?